hedera-node/hedera-app/src/main/java/com/hedera/node/app/Hedera.java

```diff
@@ -587,7 +587,7 @@ public MerkleNodeState newStateRoot() {
     @Override
     public Function<VirtualMap, MerkleNodeState> stateRootFromVirtualMap(
             @NonNull final Metrics metrics, @NonNull final Time time) {
-        return virtualMap -> new HederaVirtualMapState(virtualMap, metrics, time);
+        return virtualMap -> new VirtualMapState(virtualMap, metrics, platformStateFacade::roundOf);
```

Comment:
Just curious, why leave the `time` param in the method signature if it won't be used to construct `VirtualMapState`?